lisa/node.py

```diff
@@ -548,6 +548,7 @@ def _execute(
             update_envs=update_envs,
             command_splitter=command_splitter,
         )
+        process.check_and_input_password()
```

Comment:
It can be called in process.start now.

Reply:
yes, it can, but start function is responsible for just spawning a shell, I am not sure adding password processing there is appropriate. Does start() function need to care about password? By the way, the code change will be cleaner in process module only.

Reply:
> Does start() function need to care about password? 
It depends on how to define "start".

> By the way, the code change will be cleaner in process module only.
When the check password moved into "start", if there is another _execute called, it doesn't need to call start, check password together like this.

Reply:
The change is done, thanks